api/deploy_test.go

```diff
@@ -0,0 +1,46 @@
+package api
+
+import (
+	"kool-dev/kool/environment"
+	"testing"
+)
+
+func TestNewDeploy(t *testing.T) {
+	var tarball = "tarball"
+	d := NewDeploy(tarball)
+
+	if _, ok := d.env.(*environment.DefaultEnvStorage); !ok {
+		t.Errorf("unexpected default environment.EnvStorage")
```

Comment:
I have no idea how much extra work we have using `t.Errorf` instead of `t.Error`. In any case, I prefer using the last one when not parsing no variables into the error message.